.golangci.yml

```diff
@@ -27,19 +29,15 @@ linters:
      - structcheck
      - deadcode
      - ineffassign
+     - revive
   disable:
-    - unused #deprecated https://github.com/dominikh/go-tools/tree/master/cmd/unused
-    - gosimple #deprecated https://github.com/golangci/golangci-lint/issues/357
+    - unused #deprecated in favor of staticcheck https://github.com/dominikh/go-tools/tree/master/cmd/unused
+    - gosimple #deprecated in favor of staticcheck https://github.com/golangci/golangci-lint/issues/357
```

Comment:
Staticcheck covers `unused` and `gosimple`, but by disabling them here, it disables the `staticcheck` rules that  are covered by `gosimple` and `unused`.  (That is so that it is backwards compatible for people who do want to disable `unused` and `gosimple`). 

 I ran the linter without these disables, and looks like there are no cases of `unused`, so you can remove that line altogether. `gosimple` has a few instances, and I wonder if it would be worth fixing those while in this code.  If not, it could be another story.